### PR TITLE
Bootstrap 3 Migration: accounts/login

### PR DESCRIPTION
**Pull request by pnaw94**
scr01
scr02
scr03
scr04
scr05
@biyeun It seems everything's working fine, but I was not sure how should I migrate function-based views to properly include bootstrap3, so I've temporarily made a decorator to get it working, though I'm aware it might not be the best decision when comes to design. Do we have any standardized way of migrating that kind of views that I might have missed?

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -972,10 +973,9 @@ class HQPasswordResetForm(forms.Form):
     """
     email = forms.EmailField(label=ugettext_lazy("Username"), max_length=254)
     error_messages = {
-        'unknown': ugettext_lazy("That email address doesn't have an associated "
-                     "user account. Are you sure you've registered?"),
-        'unusable': ugettext_lazy("The user account associated with this email "
-                       "address cannot reset the password."),
+        'unknown': ugettext_lazy("That email address doesn't have an associated user account. "
+                                 "Are you sure you've registered?"),
+        'unusable': ugettext_lazy("The user account associated with this email address cannot reset the password."),
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/domain/views.py [hunk 2]
```diff
@@ -101,6 +101,7 @@
 from dimagi.utils.post import simple_post
 from toggle.models import Toggle
 from corehq.apps.hqwebapp.tasks import send_html_email_async
+from crispy_forms.utils import set_template_pack
```
Comment (dimagimon) [inline corehq/apps/domain/views.py]: F401 'set_template_pack' imported but unused

File: corehq/apps/hqwebapp/views.py [hunk 3]
```diff
@@ -44,6 +44,7 @@
 from corehq.apps.hqwebapp.encoders import LazyEncoder
 from corehq.apps.hqwebapp.forms import EmailAuthenticationForm, CloudCareAuthenticationForm
 from corehq.apps.reports.util import is_mobile_worker_with_report_access
+from corehq.apps.style.decorators import use_bootstrap3, use_bootstrap3_func
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/views.py]: F401 'use_bootstrap3' imported but unused

**Comment by biyeun** [comment]
@pnaw94 please also migrate the create account page too, thanks!

**Comment by pnaw94** [comment]
@biyeun updated to resolve conflicts
registration will be attached with another pull request

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/apps/registration/urls.py, line 4 - E261 at least two spaces before inline comment
* corehq/apps/registration/urls.py, line 7 - E261 at least two spaces before inline comment
* corehq/apps/registration/urls.py, line 7 - E501 line too long (116 > 115 characters)
* corehq/apps/registration/urls.py, line 8 - E261 at least two spaces before inline comment
* corehq/apps/registration/urls.py, line 9 - E261 at least two spaces before inline comment

**Comment by pnaw94** [comment]
@biyeun updated, along with registration module

**Comment by biyeun** [comment]
@pnaw94 conflicts

File: corehq/apps/domain/views.py [hunk 4]
```diff
@@ -23,8 +23,10 @@
 from django.contrib.auth.views import password_reset_confirm
 from django.views.decorators.http import require_POST
 from PIL import Image
-from django.utils.translation import ugettext as _, ugettext_noop, ugettext_lazy
+
+from django.utils.translation import ugettext as _, ugettext_lazy
 from django.contrib.auth.models import User
+from django.utils.translation import ugettext as _, ugettext_lazy
```
Comment (dimagimon) [inline corehq/apps/domain/views.py]: F811 redefinition of unused '_' from line 27

Comment (dimagimon) [inline corehq/apps/domain/views.py]: F811 redefinition of unused 'ugettext_lazy' from line 27

**Comment by biyeun** [comment]
Hmm, so it seems the email to new users is not being sent out properly after account creation but it's working fine on master

**Comment by biyeun** [comment]
@dimagi/product is working on a redesign of this soon, so it may be that it's not worth continuing this migration until that is settled, and we'd be doing it. It looks like with all the funciton-based views everywhere this bit looks a bit harder to migrate than I originally thought. so @pnaw94 hold off on working on this for now, let's go for the other views in the backlog. thanks!

**Pull request closed without merging**